.eslintrc.json

```diff
@@ -7,8 +7,7 @@
   "extends": [
     "airbnb",
     "prettier",
-    "prettier/react",
-    "prettier/@typescript-eslint",
+    "prettier",
```

Comment:
Dobbelt?

Reply:
Nei, kun prettier skal være igjen. Se changelog: https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21 for versjon 8.0.0

Reply:
Ja, det er bra det ble forenklet! Mente at "prettier" står to ganger i eslintrc.json:
"extends": [
    "airbnb",
    "prettier",
    "prettier", ....